Website/Resources/Search/SearchSkinObjectPreview.js

```diff
@@ -180,7 +182,9 @@
                                 href = $wrap.parent().next().attr('href');
                             }
```

Comment:
To find the $searchButton, I believe it just needs:
var $searchButton = $wrap.next();
if(!$searchButton)  $searchButton = $wrap.parent().next()